### PR TITLE
#101020 #101009 #101015 release notes: termijnbewaking

### PR DESCRIPTION
**Pull request by Klaas-Ritense**

### CODE REVIEW

File: release-notes/major12/12.0.0/valtimo-frontend-libraries.md [hunk 1]
```diff
@@ -27,9 +27,9 @@ The following features were added:
 
   More details about creating document definitions can be found [here](/using-valtimo/document/create-document-definition.md)
 
-* **Zaken-api plugin action end Hersteltermijn**
+* **Zaken-api plugin action start and end Hersteltermijn**
 
-  A Zaken-API plugin action has been added for ending a recovery period
+  A Zaken-API plugin action has been added for both starting and ending a recovery period.
```
Comment (marijnritense) [inline release-notes/major12/12.0.0/valtimo-frontend-libraries.md:32]: Can we have a link to the page where these actions are documented?

Comment (Klaas-Ritense) [reply release-notes/major12/12.0.0/valtimo-frontend-libraries.md:32]: added

File: using-valtimo/plugin/zaken-api/configure-zaken-api-plugin.md [hunk 2]
```diff
@@ -83,3 +83,17 @@ When creating a process link the following properties have to be entered:
...
+The **Start recovery period** will start a recovery period for the linked zaak in the zaken API. If a zaak has a due
+date (Uiterlijke einddatum afdoening) set, the due date will be extended with the configured maximum number of days that
+the recovery period will last. The zaak will also be suspended.
+
+- **Maximum duration in days.** The maximum duration of the recovery period in days.
+
+### End recovery period
+
+The **End recovery period** will end a recovery period for the linked zaak in the zaken API. The due
+date (Uiterlijke einddatum afdoening) of the zaak will be extended by the actual duration of the recovery period. The
```
Comment (ivo-ritense) [inline using-valtimo/plugin/zaken-api/configure-zaken-api-plugin.md:98]: Maybe we want to explain that this results in a reduction?

Comment (Klaas-Ritense) [reply using-valtimo/plugin/zaken-api/configure-zaken-api-plugin.md:98]: I've added an explaination

**Review by marijnritense — APPROVED** [review]

**Pull request merged**